### DISCUSSION TITLE
New Sonoff ZBDongle-P results in <half the LQI compared to existing Conbee II network

### DISCUSSION THREAD

**Discussion by TheQue42** [Koenkk/zigbee2mqtt#16920] [Devices]
I am in the process of migrating to a Sonoff stick, since I just noticed that I forgot to set the encryption key when migrating to z2m a few years ago, and I thought I could migrate devices one by one, instead of running around like crazy in the house for a day, re-attaching all my devices.

I started with the new Sonoff dongle on channel 20 (the old conbee, which has been *mostly* working fine, was on 15), and added a single ikea-e1812 button, just like the ones I have several of, on the old conbee/ch15 network.

But the LQI measurements are HALF of what my buttons indicate on the old network? So, I thought that, ok, maybe channel 20 wasnt such a good idea (I live in a "row-house" with zero control of all the neighbours wifi-channels), so I switched to channel 15 for the new sonoff dongle as well.

But no matter what I do, with the two ikea buttons at the same location, (and the conbee CLOSER to a unifi wifi Ap), I never get even half the LQI on the new sonoff network. (And both seem to be directly/exclusively connected to their coordinator)

All my aqara/ikea devices spread around this floor, report ~25x LQI on the conbee network, but the sonoff network gets barely 100, often less, unless I put the button NEXT to the antenna of the sonoff, then it might hit above 220....

And I have set the antenna gain to +20 db for the sonoff, but no change, in case there should have been.

Is this really what should be expected?

I have flashed the sonoff to 20221226.

Br, 
Taisto

**Comment by TheQue42** [comment]
But now I am starting to suspect that maybe I cant trust the Conbee's LQI values, since its the reciever that calculates the LQI?

https://www.sciencedirect.com/topics/engineering/link-quality
https://github.com/Koenkk/zigbee2mqtt/issues/4630 (old, but shows that the conbee has had issues with the calculation)

**Comment by Koenkk** [comment]
The Conbee II lqi computation is wrong, 100 LQI on the Sonoff is prefectly fine!

**Reply by TheQue42** [reply]
Nice to see that my suspicions were somewhat correct, and thanks for a quick reply <3 :-)

**Reply by TheQue42** [reply]
In case you have time for another quick reply....My above tests were made with to instances of z2m, starting the second with different values of base_topic and app_name.

What is the consequence of changing them, with regard to entities etc, in HA?

**Reply by Koenkk** [reply]
None, they should be updated automatically
